nifi-nar-bundles/nifi-kudu-bundle/nifi-kudu-processors/src/test/java/org/apache/nifi/processors/kudu/TestPutKudu.java

```diff
@@ -366,54 +368,219 @@ public void testUpdateFlowFiles() throws Exception {
 
     @Test
     public void testBuildRow() {
-        buildPartialRow((long) 1, "foo", (short) 10);
+        final Schema kuduSchema = new Schema(Arrays.asList(
```

Comment:
Sure, I will revert back to the abstracted method and reduce the test size.

Reply:
Since logic doesn't depend on type, I added a perimeter in the buildPartialRow method to allow you to modify the name of the id kudu column and id record field and modified this argument for unit tests.